src/components/Registration/RegAddEditOrgPerson.vue

```diff
@@ -164,6 +164,15 @@
                   a bank or a railway, use the manual entry form. All other types of business cannot
                   be {{ isProprietor ? 'the Proprietor' : 'a partner' }}.
                 </p>
+
+                <HelpContactUs class="mt-6" />
+
+                <BusinessLookup
+                  :showErrors="false"
+                  :businessLookup="EmptyBusinessLookup"
```

Comment:
Since BusinessLookup already imports some BL interfaces, why not just import this one and remove it as a prop?

Reply:
I have to update this to support selected data (like summary) in the next ticket. So it need to be a prop